demo/index.js

```diff
@@ -18,3 +18,23 @@ createFocusGroup({
 	},
 	wrap: true,
 }).activate();
+
+var selectRoot = document.getElementById('select-root');
+var selectNodes = document.querySelectorAll('.select');
+var focusGroup = createFocusGroup({
+  rootNode: selectRoot,
+	members: selectNodes,
+	stringSearch: true,
+	keybindings: {
+		next: [{ keyCode: 40 }, { keyCode: 39 }],
+		prev: [{ keyCode: 38 }, { keyCode: 37 }]
+	},
+	wrap: true,
+}).activate();
+
+focusGroup.on('focus', function (member) {
```

Comment:
I don't like that there's a `'focus'` event that is not actually a (native) `'focus'` event. How about making this a `'move'` event, or something like that?